src/shared/services/map-location/index.js

```diff
@@ -70,28 +78,14 @@ export const formatMapLocation = location => {
   }
 
   if (location.address) {
-    const { openbare_ruimte, huisnummer, postcode, woonplaats } = location.address;
-    const addressText = [
-      [openbare_ruimte, huisnummer],
-      [postcode, woonplaats],
-    ]
-      .flatMap(parts => parts.filter(Boolean).join(' '))
-      .join(', ');
-
-    value.addressText = addressText;
+    value.addressText = getAddressText(location.address);
     value.address = location.address;
   }
 
   return value;
 };
 
-export const formatAddress = address => {
-  const toevoeging = address.huisnummer_toevoeging ? `-${address.huisnummer_toevoeging}` : '';
-  const display = address.openbare_ruimte
-    ? `${address.openbare_ruimte} ${address.huisnummer}${address.huisletter}${toevoeging}, ${address.postcode} ${address.woonplaats}`
-    : '';
-  return display;
-};
+export const formatAddress = address => getAddressText(address);
```

Comment:
Can this function been dropped?

Reply:
It's used somewhere else and I didn't want to touch that part of the application just yet.